fotoapparat/src/main/java/io/fotoapparat/parameter/Parameters.java

```diff
@@ -29,7 +29,8 @@ public void putValue(Type type, Object value) {
 
     private void ensureType(Type type, Object value) {
         if (value == null) {
-            return;
+            throw new IllegalArgumentException("Provided " + type.clazz.getSimpleName()
```

Comment:
The idea behind `Parameters` is that it essentially behaves as a map and map does allow `null` values to be written. Moreover, we should always expect `null` values to be returned from `getValue` anyway, so this check does not really protects us from anything.